killercoda-sample/karmada-CLI-installtion-example/intro.md

```diff
@@ -0,0 +1,44 @@
+### 环境背景：
+
+1. 退出node01机器
+
+   ```shell
+      exit
```

Comment:
Do I have to enter the `exit` command in order to go back to the master node?

Reply:
Instead of logging in through ssh, I use the following code to avoid the need to log out from node01：
```shell
# create cluster in node01 machine
ssh root@${member_cluster_ip} "bash ~/installKind.sh" &
sleep 10
ssh root@${member_cluster_ip} "bash ~/createCluster.sh"
```